Modules/+Sources/@VelocityLaser/VelocityLaser.m

```diff
@@ -41,19 +41,33 @@
-        diode_on = Prefs.Boolean(false,'set','set_diode_on','help','Power state of diode (on/off)');
-        wavemeter_active = Prefs.Boolean(false,'set','set_wavemeter_active','help','Wavemeter channel active');
-        percent_setpoint = Prefs.Double(NaN,'units','%','help','local memory of tuning percent as applied by the wavemeter');
-        TuneSetpointAttempts = Prefs.Integer(3,'min',1,'allow_nan',false);
-        TuneSetpointNPoints = Prefs.Integer(25,'min',1,'allow_nan',false,'help','number of wavemeter queries below wavemeter resolution to consider settled.');
+        tuning =                Prefs.Boolean(false,'readonly',true);
+        debug =                 Prefs.Boolean(false);
+        TuningTimeout =         Prefs.Double(60,'units','sec','min',0,'help','Timeout for home-built PID used in TuneCoarse');
+        
+        pb_ip =                 Prefs.String('No Server','set','set_pb_ip','help','IP/hostname of computer with PB server');
+        PBline =                Prefs.Integer(12,'min',1,'allow_nan',false,'set','set_PBline','help','Indexed from 1');
+        
+        velocity_ip =           Prefs.String('No Server','set','set_velocity_ip','help','IP/hostname of computer with hwserver for velocity laser');
+        
+        wavemeter_ip =          Prefs.String('No Server','set','set_wavemeter_ip','help','IP/hostname of computer with hwserver for wavemeter');
+        wavemeter_channel =     Prefs.Integer(3,'min',1,'allow_nan',false,'set','set_wavemeter_channel','help','Pulse Blaster flag bit (indexed from 1)');
+        
+        wheel_ip =              Prefs.String('No Server', 'set','set_wheel_ip', 'help', 'IP/hostname of computer with hwserver for Arduino-controlled filter wheel.');
+        wheel_pin =             Prefs.Integer(2, 'min',2, 'max', 13, 'allow_nan', false, 'set', 'set_wheel_pin', 'help', 'Pin on the Arduino corresponding to the filter wheel servo.');
+        wheel_pos =             Prefs.MultipleChoice('OD0', 'choices', Sources.VelocityLaser.wheel_choices, 'allow_empty', false, 'set' ,'set_wheel_pos' ,'help', 'Current position of the Arduino-controlled filter wheel. The wheel weaves as the wheel wills.');
+        
+        diode_on =              Prefs.Boolean(false,'set','set_diode_on','help','Power state of diode (on/off)');
+        
+        wavemeter_active =      Prefs.Boolean(false,'set','set_wavemeter_active','help','Wavemeter channel active');
+        
+        percent_setpoint =      Prefs.Double(NaN,'units','%','help','local memory of tuning percent as applied by the wavemeter');
+        
+        TuneSetpointAttempts =  Prefs.Integer(3,'min',1,'allow_nan',false);
+        TuneSetpointNPoints =   Prefs.Integer(25,'min',1,'allow_nan',false,'help','number of wavemeter queries below wavemeter resolution to consider settled.');
+    end
+    properties(Constant)
+        wheel_choices = {'OD.5', 'OD1', 'OD0', 'OD2'};
```

Comment:
What's the best way for us to in general implement things like this? This is a constant for this specific hardware instance and will likely be immutable once installed, but it could be different for different physical systems that call the same driver. Can this be set similar to how we do the nidaq lines? @mwalsh161

Reply:
ya i was thinking about commenting here too. I was going to suggest that this live on hwserver config, but then that didn't make sense because the hwserver only cares about the angle of the servo. However, I think you're right, this could behave just like nidaq on the client side and define what the different positions of the servo actually mean.